plugins/outputs/influxdb_v2/http.go

```diff
@@ -75,14 +75,6 @@ func (c *httpClient) Init() error {
 		c.headers = make(map[string]string, 2)
```

Comment:
This should be
```suggestion
		c.headers = make(map[string]string, 1)
```
shouldn't it?